src/LightClient.ts

```diff
@@ -0,0 +1,25 @@
+export class LightClient {
+  shardName: string;
+
+  constructor(shardName: string)
+  {
+    this.shardName = shardName;
+  }
+
+  async getCurrentBlockHeight(): Promise<bigint> {
+    const {providers} = require("near-api-js");
+    const provider = new providers.JsonRpcProvider(
+      "https://rpc.testnet.near.org"
+    );
+
+    const rawResult = await provider.query({
+      request_type: "call_function",
+      account_id: "light_client." + this.shardName + ".dev.calimero.testnet",
```

Comment:
consider putting all of these hardcoded strings in a consts file, since some of those might change later on.
e.g.
```
const light_client_contract_prefix = "light_client";
const calimero_testnet_account_suffix = "dev.calimero.testnet";
```

Reply:
Not sure how relevant this is, but `dev.calimero.testnet` is used only on dev environment.
On staging it's `stage.calimero.testnet` and on prod it's just `calimero.testnet`.

Reply:
@isoldo good point, another param to the constructor might be the env

Reply:
Ok, will add this to constants and environment to the constructor when I start refactoring this code. For now I am just adding functionalities to the SDK